info/available-rules.md

```diff
@@ -61,3 +61,4 @@
 |    3 |      3.8 | STRING_CONCATENATION                    | Check: warns if in a single line concatenation of strings is used                                                                                                            | yes | StringConcatenationWarnTest |- | -
 |    3 |      3.6 | TOO_MANY_BLANK_LINES                    | Check: warns if blank lines are used placed incorrectly.<br>Fix: removes redundant blank lines.                                                                              | yes | BlankLinesWarnTest<br>BlankLinesFixTest                                                                                                                                                                    |                                                                                    - |              
 |    3 |      3.7 | WRONG_WHITESPACE                        | Check: warns if usage of horizontal spaces violates code style guide.<br>Fix: fixes incorrect whitespaces.                                                                   | yes | WhiteSpaceRuleWarnTest<br>WhiteSpaceRuleFixTest                                                                                                                                                            |                         - |                                                                                     - |
+|    3 |      3.8 | TOO_MANY_CONSECUTIVE_SPACES
```

Comment:
Please add `|` for empty columns too, also please fill description and configuration column, they are important, especially since your warning has some configuration options.